### ISSUE TITLE
Segmentation fault on macOS 12.4 (Apple M1)

### ISSUE THREAD

**Issue by petoem** [Orama-Interactive/Pixelorama#718]
**Pixelorama version:** v0.10.1

**OS/device including version:** macOS 12.4 (Apple M1)

**Issue description:** Crashes immediately when run. 
```
Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000010
Exception Codes:       0x0000000000000001, 0x0000000000000010
Exception Note:        EXC_CORPSE_NOTIFY

Termination Reason:    Namespace SIGNAL, Code 11 Segmentation fault: 11
Terminating Process:   exc handler [54798]
```

**Steps to reproduce:**Open Pixelorama.

**Comment by jferdelyi** [comment]
The same here:

- Pixelorama v0.10.1
- macOS 12.4
- Apple Silicon (ARM based) M1 Pro processor
- Immediate crash when opening

> /Applications/Pixelorama.app/Contents/MacOS/Pixelorama ; exit;
> arguments
> 0: /Applications/Pixelorama.app/Contents/MacOS/Pixelorama
> Godot Engine v3.4.2.stable.official.45eaa2daf - https://godotengine.org
> OpenGL ES 2.0 Renderer: Apple M1 Pro
> OpenGL ES Batching: ON
>  
> UNSUPPORTED (log once): POSSIBLE ISSUE: unit 1 GLD_TEXTURE_INDEX_2D is unloadable and bound to sampler type (Float) - using zero texture because texture unloadable
> [1]    36677 segmentation fault  /Applications/Pixelorama.app/Contents/MacOS/Pixelorama
> 
> Saving session...completed.
>

**Comment by OverloadedOrama** [comment]
Can you check if it also crashes on v0.10.2-dev? https://nightly.link/Orama-Interactive/Pixelorama/workflows/dev-desktop-builds/master

**Comment by jeremyk6** [comment]
Hello. I also had this crash on 0.10.1. It doesn't happen with 0.10.2-dev, that launches flawlessly !

**Comment by petoem** [comment]
@OverloadedOrama The (latest) build now runs. 
https://github.com/Orama-Interactive/Pixelorama/actions/runs/2704683416

**Comment by jferdelyi** [comment]
It doesn't happen with 0.10.2-dev too !

**Comment by OverloadedOrama** [comment]
I forgot to mention that I published a hotfix for 0.10.1 some days ago, it should no longer crash now if you re-download it.
